documentation/disaster-recovery-testing.md

```diff
@@ -0,0 +1,176 @@
+# Disaster Recovery testing
+
+This document covers the Disaster Recovery testing procedure for applications hosted on the Teacher Services AKS clusters.
+
+## Prerequisites
+
+- Identified environment for the test e.g. qa, staging, test, etc
+- Identified scenario(s) that are to be tested
+    1. loss of database instance
+    2. loss of data
+    3. loss of other service resources e.g. deployment
+        - this can be tested alongside option 1 or 2?
+- Repo workflows that should utilise existing DFE github-actions
+    - deploy selected env, and can pass send_traffic_to_maintenance_page:true to deploy [required for scenarios 1,3 above]
+        - e.g. https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/.github/workflows/deploy-v2.yml
+    - backup postgres database to Azure storage [required for scenario 1 above]
+        - using https://github.com/DFE-Digital/github-actions/tree/master/backup-postgres
+    - restore postgres database from Azure storage [required for scenario 1 above]
+        - using https://github.com/DFE-Digital/github-actions/tree/master/restore-postgres-backup
+    - point in time restore of postgres database [required for scenario 2 above]
+        - using tba
+- Repo workflows to enable and disable the maintenance page.
+    - see https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/maintenance-page.md
+    - confirm workflows exists for the selected environment to be tested. Examples;
+        - https://github.com/DFE-Digital/apply-for-teacher-training/actions/workflows/enable-maintenance.yml
+        - https://github.com/DFE-Digital/apply-for-teacher-training/actions/workflows/disable-maintenance.yml
+
+## Documentation requirements
+
+Copy the template DR testing document which will be a record of the steps run, time taken, and any issues
+- template doc tba
+
+## Initial set-up
+
+There are no local dependencies to complete the DR test.
+Schedule virtual meeting for the test to take place
+- teams or slack
+- invite relevant technical stakeholders
+Regularly provide updates on the service Slack channel to keep product owners abreast of developments.
+
+## Freeze pipeline
+
+Alert developers that no one should merge to main.
+- In github setings, update the Branch protection rules and set required PR approvers to 6
+
+## Disable statuscake monitoring
```

Comment:
It may be interesting to keep it up to validate alerting

Reply:
removed statuscake disable/enable